containers/DvmState.ts

```diff
@@ -12,24 +14,19 @@ const { formatUnits: fromWei } = utils;
 interface ContractState {
   hasEmpPrice: boolean | null;
   resolvedPrice: number | null;
+  finalFee: number | null;
 }
 
 const initState = {
   hasEmpPrice: null,
   resolvedPrice: null,
+  finalFee: null,
 };
 
-// Taken from Voting.sol implementation.
```

Comment:
this was left over from when we were using theGraph to do some of these queries. I removed it as it's not needed any more.